src/Valet/App.cs

```diff
@@ -74,6 +74,29 @@ public async Task<int> GetVersionAsync()
         return 0;
     }
 
+    public async Task CheckForUpdatesAsync()
+    {
+        try
+        {
+            var latestImageDigestTask = _dockerService.GetLatestImageDigestAsync(ValetImage, ValetContainerRegistry);
+            var currentImageDigestTask = _dockerService.GetCurrentImageDigestAsync(ValetImage, ValetContainerRegistry);
+
+            await Task.WhenAll(latestImageDigestTask, currentImageDigestTask);
+
+            var latestImageDigest = latestImageDigestTask?.Result;
+            var currentImageDigest = currentImageDigestTask?.Result;
+
+            if (latestImageDigest != null && currentImageDigest != null && !latestImageDigest.Equals(currentImageDigest))
+            {
+                Console.WriteLine("A new version of the Valet CLI is available. Run 'gh valet update' to update.");
+            }
+        }
+        catch (Exception)
+        {
+            // Ignore for now
```

Comment:
Should we update the comment stating why we are ignoring?  Mainly for the reason of a future developer having a easier time understanding the reasoning